proxy/proxy.go

```diff
@@ -829,6 +867,14 @@ func (p *Proxy) makeBackendRequest(ctx *context, requestContext stdlibcontext.Co
 	if endpoint != nil {
 		endpoint.Metrics.IncInflightRequest()
 		defer endpoint.Metrics.DecInflightRequest()
+
+		_, h, err := normalizeSchemeHost(endpoint.Scheme, endpoint.Host)
```

Comment:
This should not happen on the request path. Either during route creation or not at all.